src/content/recipes/vuetify.md

```diff
@@ -0,0 +1,196 @@
+<div class="aside aside__no-top">
+
+This recipe assumes that you have a Vue 3 app using Vuetify v3 and have just set up Storybook 7.0 using the [getting started guide](https://storybook.js.org/docs/7.0/vue/get-started/install). Don’t have this? Follow Vuetify’s [installation instructions](https://next.vuetifyjs.com/en/getting-started/installation/#installation) then run:
```

Comment:
Does that link have to use the absolute URL? If it's relative, it should take advantage of GatsbyLink's client-side routing.

```suggestion
This recipe assumes that you have a Vue 3 app using Vuetify v3 and have just set up Storybook 7.0 using the [getting started guide](/docs/7.0/vue/get-started/install). Don’t have this? Follow Vuetify’s [installation instructions](https://next.vuetifyjs.com/en/getting-started/installation/#installation) then run:
```